lib/src/values/logic_value.dart

```diff
@@ -881,7 +882,7 @@ abstract class LogicValue implements Comparable<LogicValue> {
     }
     if (this is BigInt ||
         this is _BigLogicValue ||
-        (this is _FilledLogicValue && width >= _INT_BITS)) {
+        (this is _FilledLogicValue && width > _INT_BITS)) {
```

Comment:
I didn't notice before that you caught and fixed this one too, thanks! We should add some unit test that hits the unary math on these borders as well?

Reply:
unary math?  Can you give an example?
I'm looking at the divide issue as well.

Reply:
You can see the complete set of things that use it by finding "find all references" to `_doUnaryMath`, which I'm surprised to find (forgot) is only the `clog2` function.  It looks like there's a lot of tests already in "logic_value_test.dart", I dont know if the existing tests cover the types of scenarios being addressed in this PR though.  The other "unary" math like and/or/xor reduction use their own custom implementations per-flavor already.